### ISSUE TITLE
Left cleft lip and palate

### ISSUE THREAD

**Issue by taylordm** [obophenotype/human-phenotype-ontology#7135]
**Preferred term label:**
Left cleft lip and palate 

**Synonyms**

**Definition (free text, please give PubMed ID)**

Kids First dx: left (non-midline) cleft lip AND palate together as one phenotype

**Parent term (use hpo.jax.org/app)**
Cleft lip & palate  HP:0008501 
but this is non-midline. I'm assuing the parent Cleft lip & palate are assumed to be midline? Or do we need under cleft lip & palate: midline cleft lip & palate; bilateral cleft lip & palate, and left cleft lip & palate, and right cleft lip & palate

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Please use the modifier term for left https://hpo.jax.org/app/browse/term/HP:0012835
In general, the HPO does not make subterms for right and left unless there is a clinical difference (which I think is **very** rarely the case).
I would also recommend using separate annotations for lip and palate.
